src/equations/compressible_navier_stokes_1d.jl

```diff
@@ -159,10 +160,15 @@ function flux(u, gradients, orientation::Integer,
     # in the implementation
     q1 = equations.kappa * dTdx
 
-    # Constant dynamic viscosity is copied to a variable for readability.
-    # Offers flexibility for dynamic viscosity via Sutherland's law where it depends
-    # on temperature and reference values, Ts and Tref such that mu(T)
-    mu = equations.mu
+    # In the simplest case, `mu(u, equations)` returns just a constant but
+    # more complex functions like Sutherland's law are possible.
```

Comment:
```suggestion
    # In the simplest case, `mu` is just a constant, but
    # more complex functions like Sutherland's law are possible.
```
This also applies to the 2D and 3D version.